# Задача №1

Вероятность того, что стрелок попадет в мишень, выстрелив один раз, равна 0.8. Стрелок выстрелил 100 раз. Найдите вероятность того, что стрелок попадет в цель ровно 85 раз.

Для решения этой задачи нам понадобится формула Бернулли:

$$P_n(X=k) = C_n^k p^k q^{n-k}$$

при $k = 85$, $n = 100$ и $p = 0.8$.

$$P_{100}(X=85) = C_{100}^{85} * 0.8^{85} * 0.2^{15}$$

In [ ]:
from math import factorial

def combinations(n, k):
    return int(factorial(n) / (factorial(k) * factorial(n - k)))

In [ ]:
n = 100
k = 85
p = 0.8
X = combinations(n, k) * (p ** k) * (1 - p) ** (n - k)
X

0.048061793700746355

## **Ответ:** $P_{85} = 0.04806$

# Задача №2

Вероятность того, что лампочка перегорит в течение первого дня эксплуатации, равна 0.0004. В жилом комплексе после ремонта в один день включили 5000 новых лампочек. 
1. Какова вероятность, что ни одна из них не перегорит в первый день? 
2. Какова вероятность, что перегорят ровно две?

## 2.1 Какова вероятность, что ни одна из них не перегорит в первый день?

Для решения этой задачи необходимо воспользоваться формулой Пуассона

$$P_m \approx \frac{(np)^m}{m!}e^{-np}$$

при $p = 0.0004$, $n = 5000$ и $m = 0$.

$$P_{0} \approx \frac{(5000 * 0.0004)^{0}}{0!}e^{-5000 * 0.0004} = 
e^{-2}$$

In [ ]:
np.exp(-2)

0.1353352832366127

### **Ответ 2.1:** $P_{0} \approx 0.1353$

## 2.2 Какова вероятность, что перегорят ровно две?

В этом случае нам так же пеобходима формула Пуассона.

$p = 0.0004, n = 5000, m = 2$

$$P_2 \approx \frac{(5000 * 0.0004)^{2}}{2!}e^{-5000 * 0.0004} = \frac{2^2}{2}e^{-2} = 2 * P_0 \approx 0.2706$$

### **Ответ 2.1:** $P_2 \approx 0.2706$

# Задача №3

Монету подбросили 144 раза. Какова вероятность, что орел выпадет ровно 70 раз?

$$ P = C_{144}^{70} * 0.5^{70} * 0.5^{74}$$

In [ ]:
n = 144
k = 70
p = 0.5
X = combinations(n, k) * (p ** k) * (1 - p) ** (n - k)
X

0.06281178035144776

## **Ответ:** $P = 0.06281$

# Задача №4

В первом ящике находится 10 мячей, из которых 7 - белые. Во втором ящике - 11 мячей, из которых 9 белых. Из каждого ящика вытаскивают случайным образом по два мяча. 
1. Какова вероятность того, что все мячи белые? 
2. Какова вероятность того, что ровно два мяча белые? 
3. Какова вероятность того, что хотя бы один мяч белый?

## 4.1 Какова вероятность того, что все мячи белые?

Искомая вероятность равна вероятности того, что из первого и из второго ящика вытащили по 2 белых мяча. Т.е.

$$P = P_1(X = 2) * P_2(X = 2)$$

$$P_1 = C_2^2 * 0.7^2 * 0.3^0 = 0.14$$

$$P_2 = C_2^2 * (\frac{9}{11})^2 = \frac{81}{121}$$

Итак, $P = 0.14 * \frac{81}{121}$

In [26]:
0.14 * 81 / 121

0.09371900826446282

### **Ответ:** $P \approx 0.0937$

## 4.2 Какова вероятность того, что ровно два мяча белые?

$$P = P_{1,2}P_{2,0} + P_{1,1}P_{2,1} + P_{1,0}P_{2,2}$$
$$P_{1,2} = 0.14$$
$$P_{2,0} = \frac{4}{121}$$
$$P_{1,1} = C_2^1 * 0.7 * 0.3 = 0.42$$
$$P_{2,1} = C_2^1 * \frac{18}{121} = \frac{36}{121}$$
$$P_{1,0} = 0.09$$
$$P_{2,2} = \frac{81}{121}$$

Здесь первый индекс у $P$ - это номер ящика, а второй - количество вытащенных белых мячей.
 
В итоге 

$$P = \frac{0.14 * 4}{121} + \frac{0.42 * 36}{121} + \frac{0.09 * 81}{121}$$

In [27]:
0.14 * 4 / 121 + 0.42 * 36 / 121 + 0.09 * 81 / 121

0.18983471074380165

### **Ответ:** $P \approx 0.1898$

## 4.3 Какова вероятность того, что хотя бы один мяч белый?

$$P = 1 - P_{1,0} * P_{2,0}$$

Мы уже выяснили в 4.2, что $P_{1,0} = 0.09$ и $P_{2,0} = \frac{4}{121}$

Следовательно
$$P = 1 - \frac{0.09 * 4}{121}$$

In [28]:
1 - 0.09 * 4 / 121

0.9970247933884298

### **Ответ:** $P \approx 0.997$